services/apps/src/apps/routes/domain/permission.clj

```diff
@@ -0,0 +1,18 @@
+(ns apps.routes.domain.permission
+  (:use [common-swagger-api.schema :only [describe]]
+        [schema.core :only [defschema optional-key enum]])
+  (:import [java.util UUID]))
+
+(defschema AppIdList
+  {:apps (describe [String] "A List of app identifiers")})
+
+(defschema UserPermission
+  {:user       (describe String "The user ID")
```

Comment:
I'd have this be `NonBlankString` from `common-swagger-api.schema` instead (same for other strings that shouldn't be blank), just for some extra validation/documentation.